### ISSUE TITLE
city-scrapers-core module not found

### ISSUE THREAD

**Issue by nathanderon** [bonfirefan/city-scrapers-pitt#60]

**Comment by ChrisJabb21** [comment]
![ModuleNotFoundErrorCapture](https://user-images.githubusercontent.com/12942076/66245614-05bdda80-e6dd-11e9-8803-5aa8b447b528.PNG)

**Comment by bonfirefan** [comment]
try uninstalling and reinstalling `city-scrapers-core` with `pipenv install city-scrapers-core`

**Comment by bonfirefan** [comment]
also make sure this branch is synced with master
Scroll to the Git section here on syncing the fork: https://cityscrapers.org/docs/setup-help/
Use these commands (don't sync with the main City Bureau project):
`git remote add upstream https://github.com/bonfirefan/city-scrapers-pitt.git`
Then follow the steps as listed

**Comment by ChrisJabb21** [comment]
Thanks Bonnie, that did the trick. I didn't have the city-scraper-core library installed and had to install it. I am just having trouble with pushing the work Nick did that I tested on my machine to my github repo. I will keep trying to see if i can get it. I added the upstream, but I am not sure about the venv commands from the link you sent me. 
![image](https://user-images.githubusercontent.com/12942076/66278320-ef975200-e875-11e9-894a-120a293c36ba.png)
